app/Http/Controllers/Service/RegistrationController.php

```diff
@@ -251,7 +252,7 @@ function ($child) {
 
     public function update(StoreUpdateRegistrationRequest $request)
     {
-        // TODO: add validation on the request like store has.
+        $user = $request->user();
```

Comment:
Should it be `Auth::user();` ?

Reply:
Both result in the identical thing according to the docs. I thought i'd use $request, since it was injected already.